src/experimental/information_estimates.cpp

```diff
@@ -100,29 +98,105 @@ std::tuple<int, kt_double> InformationEstimates::findLessInformativeLaser(std::v
 
                 // Compute all the possible combinations for the current cell - algorithm 1
                 std::unordered_map<map_tuple, kt_double, utils::tuple_hash::HashTuple> meas_out_prob = computeMeasurementOutcomesHistogram(cell_prob);
-                
+
                 kt_double cell_mutual_inf = 0.0f;
                 for (auto& pair : meas_out_prob)
                 {
                     cell_mutual_inf +=  pair.second * measurementOutcomeEntropy(pair.first);
                 }
 
-                // Mutual information of cell x, y given a set of measurements                
-                updateCellMutualInformation(1.0 - cell_mutual_inf, cell); 
+                // Mutual information of cell x, y given a set of measurements
+                updateCellMutualInformation(1.0 - cell_mutual_inf, cell);
             }
         }
-        // Mutual information provided by this laser scan
-        kt_double laser_mut_info = calculateLaserMutualInformation();
-        scans_mut_info.emplace_back(laser_mut_info);
-        
-        utils::grid_operations::clearVisitedCells(m_info_grid);
     }
-
-    kt_double map_mut_info = calculateMapMutualInformation();
+    // Total mutual information
+    kt_double map_mut_info = m_mutual_grid.sum();
 
     // Clearing the cells for the next time it is called
     utils::grid_operations::clearVisitedCells(m_mutual_grid);
 
+    for (int i = 0; i < range_scans.size(); ++i)
```

Comment:
@kmilo7204 nit: can we do better than outright copying and pasting pretty much the same code twice?